src/Broadway/AMQP/EventBusForwardingConsumerFactory.php

```diff
@@ -71,15 +62,10 @@ public function __construct(
         $this->consumerTag = $consumerTag;
     }
 
-    /**
-     * @param StringLiteral $exchange
-     * @param StringLiteral $queue
-     * @return EventBusForwardingConsumer
-     */
     public function create(
         StringLiteral $exchange,
         StringLiteral $queue
-    ) {
+    ):EventBusForwardingConsumer {
```

Comment:
Super small nit: Missing a space here between `:` and `EventBusForwardingConsumer` 😛

Reply:
Fixed